src/components/MoneyRequestAmountInput.tsx

```diff
@@ -279,6 +284,7 @@ function MoneyRequestAmountInput(
             prefixContainerStyle={props.prefixContainerStyle}
             touchableInputWrapperStyle={props.touchableInputWrapperStyle}
             maxLength={maxLength}
+            hideFocusedState={hideFocusedState}
```

Comment:
@youssef-lr Why did we introduce this new prop? I'm not sure if I understand what this is doing.

Reply:
It was always `true` by default. We're reusing the amount input here and we want it to be configurable as we'd like to show the focused state in the amounts used in the split page.

Reply:
See here https://github.com/Expensify/App/blob/936bb9363caa83dd2bd7635743a90ec59dc0c7c2/src/components/AmountTextInput.tsx#L64

Reply:
Ah, right. I see it now. Thanks!